pkg/api/v20210131preview/openshiftcluster_validatestatic_test.go

```diff
@@ -456,10 +466,31 @@ func TestOpenShiftClusterStaticValidateNetworkProfile(t *testing.T) {
 			},
 			wantErr: "400: InvalidParameter: properties.networkProfile.serviceCidr: The provided vnet CIDR '10.0.0.0/23' is invalid: must be /22 or larger.",
 		},
+		{
+			name: "sdnProvider given as empty",
+			modify: func(oc *OpenShiftCluster) {
+				oc.Properties.NetworkProfile.SDNProvider = ""
+			},
+			wantErr: "400: InvalidParameter: properties.networkProfile.sdnProvider: The provided SDNProvider '' is invalid.",
+		},
+		{
+			name: "sdnProvider defaults as OVNKubernetes, no change",
+			modify: func(oc *OpenShiftCluster) {
+				oc.Properties.NetworkProfile.SDNProvider = "OVNKubernetes"
+			},
+		},
```

Comment:
I think we should drop this test: I believe it does not test anything.

Reply:
https://github.com/Azure/ARO-RP/pull/1571/commits/7cf26fefa84635caced03ddb15651f0635027f4f#diff-d2a80e56830699a55c16cd21471ad6f85b8061d822f37b247f4b5021f337dfe0L476-L481 

Removed.